core-tests/shared/src/test/scala/zio/ZIOSpec.scala

```diff
@@ -2284,7 +2284,7 @@ object ZIOSpec extends ZIOBaseSpec {
       testM("deep fork/join identity") {
```

Comment:
Not sure why this test isn't working on Scala.js. It completes normally if I run the tests in parallel but fails with out of memory if I run them sequentially. Maybe something about starting with a fresh fiber?

Reply:
Could test hypothesis with `@@ forked` to further narrow down. Good job locating the problem to this test though.

Reply:
Still seeing the same issue with forking both that test and all tests in the suite so still not sure what is going on here.